fm4tlp/test.py

```diff
@@ -249,7 +255,7 @@ def main(_):
       total_nodes, size=_NUM_NEIGHBORS.value, device=device
   )
 
-  model_config = model_config_lib.get_model_config(_MODEL_NAME.value)
+  model_config = experiment_config.get_model_config(_MODEL_NAME.value)
```

Comment:
As far as I can tell, [experiment_config.py](https://github.com/google-research/google-research/blob/master/fm4tlp/experiment_config.py) does not contain a `get_model_config` function. Why is this change made?

Reply:
We removed model_config.py and moved its functions/content to experiment_config.py 

I missed adding this change into this pull request.

Reply:
We replaced model_config.py with experiment_config.py
I missed adding this to the initial PR request, will fix in next version.

Reply:
Thanks for the clarification. Is there a reason to merge the two files that goes beyond code organization preference? I would prefer not to remove model_config unless there is a specific reason that we can point to. Thanks